### ISSUE TITLE
I am not able to complete last step of the module 2

### ISSUE THREAD

**Issue by Parnilay** [insidesherpa/JPMC-tech-task-2#187]
i have done all the steps as per given to setup environment for the program. I just didnt install node.js  as i have already installed it before for other uses and it is having latest version and same goes for npm. Now, i installed npm in this folder after it everyhting goes good and after clicking the button of "Start Streaming Data" on the page an error appears. I am attaching all the possible screenshots of the error and issue related to it. Please Help!!
![Screenshot (30)](https://user-images.githubusercontent.com/40804561/83196486-9d36d880-a159-11ea-856e-07abac757040.png)
![Screenshot (31)](https://user-images.githubusercontent.com/40804561/83196496-a2942300-a159-11ea-94ab-b9232490902f.png)
![Screenshot (32)](https://user-images.githubusercontent.com/40804561/83196506-a6c04080-a159-11ea-9b3e-2ac1a616853e.png)
![Screenshot (33)](https://user-images.githubusercontent.com/40804561/83196514-aa53c780-a159-11ea-8938-6c8fe493fd51.png)

**Comment by Harshada0295** [comment]
Hello,I am also facing same issue did you find any solution for this ?

**Comment by Parnilay** [comment]
No, I didn't found any solution. If you get one, do share it with me.

**Comment by diggty66** [comment]
Do you have python datafeed/server.py running in separate window?
